draft-bbs-signatures.md

```diff
@@ -694,15 +694,15 @@ The ZKP protocols use nonces which MUST be different in each context.
 
 ## Choice of underlying curve
 
-BBS signatures can be implemented on any pairing-friendly curve. However care MUST be taken when selecting one that is appropriate, this specification defines a profile for using the BLS12-381 curve in (#ciphersuites) which as a curve currently achieves close to 128-bit security.
+BBS signatures can be implemented on any pairing-friendly curve. However care MUST be taken when selecting one that is appropriate, this specification defines a ciphersuite for using the BLS12-381 curve in (#ciphersuites) which as a curve currently achieves close to 128-bit security.
```

Comment:
We could reference http://www.csl.sri.com/papers/vcdm-did-crypto-recs/crypto-review-and-recs-for-VCDM-and-DIDs-implems-FINAL-20211015.pdf here, estimating the strength at 117-120 bits. It's definitely less than 128 bits, but still "virtually one million times stronger than BN-254"

Reply:
Thanks have updated, what do you think?

Reply:
Good enough for me :)